### ISSUE TITLE
Channels assignment for setBroadvastFreq

### ISSUE THREAD

**Issue by sytelus** [dji-sdk/Onboard-SDK#49]
It looks like below function allows to set broadcast frequency for data. However it takes array of size 16, each of which elements probably mean certain broadcast data type. This doesn't seem to be documented anywhere. What is the default channel assignment for Matrice 100 for below function? I want to set broadcast frequency for specific messages (for example, local_position) and set everything else to 0 Hz to remove noise. I can certainly do this from PC Assistant but it would be great to be able to do from APIs at run time. Also when can below function may be called (after activation or after RequestControl)?

`void CoreAPI::setBroadcastFreq(uint8_t *dataLenIs16, CallBack callback, UserData userData)`

It would be also nice to update the API doc with this info.

**Comment by lanyusea** [comment]
https://github.com/dji-sdk/Onboard-SDK/blob/3.1/doc/en/OPENProtocol.md

please refer to command 0x00 0x10

**Comment by sytelus** [comment]
Thank you! Just wondering if there are any restrictions on when this command can be executed...

**Comment by lanyusea** [comment]
@sytelus should be no restriction. As I remember, it is the same as the one in DJI Assistant, by which you can modify the frequency whenever you want.

**Comment by peza8** [comment]
Hi @lanyusea - This topic was closed on 23 May 2016, and the reference to the OPEN protocol doesn't seem to match the 'Available Broadcast Data' described in the DJI documentation [here](https://developer.dji.com/onboard-sdk/documentation/guides/component-guide-telemetry.html). 

I have an M210 and am interested in accessing Relative Position. Where can I set the broadcast frequency of this topic? Thanks.
